### ISSUE TITLE
Fails to build in Xcode 10: kCAFillModeForwards

### ISSUE THREAD

**Issue by abdulla-allaith** [egmoll7/EMSpinnerButton#6]
There's only one error causing the library to fail building in Xcode 10:

```
Pods/EMSpinnerButton/EMSpinnerButton/EMSpinnerButton/EMSpinnerButton.swift:136:25: 'kCAFillModeForwards' has been renamed to 'CAMediaTimingFillMode.forwards'
was obsolete in Swift 3
```

Please fix this so the library can be built with Xcode 10 🙏

**Comment by omidgolparvar** [comment]
@abdulla-allaith You can set `Swift Language Version` for this `pod` and temporary solve the problem. 

<img width="1399" alt="screen shot 2018-11-01 at 01 02 20" src="https://user-images.githubusercontent.com/5128342/47820149-e3f00100-dd71-11e8-950c-c407bc276413.png">

**Comment by egmoll7** [comment]
This issue has been resolved in the latest version (1.0.1)
